SAMtags.tex

```diff
@@ -246,6 +250,39 @@ \subsection{Barcodes}
 The {\tt BC} tag should match the {\tt QT} tag in length. 
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and places a hyphen (`{\tt -}') between the barcodes from the same template. 
 
+\item[BZ:Z:\tagvalue{qualities+}] 
+Phred quality of the (uncorrected) unique molecular identifier sequence in the {\tt OX} tag.
+Same encoding as {\sf QUAL}, i.e., Phred score + 33.
+The {\tt OX} tags should match the {\tt BZ} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with a space (`{\tt \textvisiblespace}') between the different strings.
+
+\item[CB:Z:\tagvalue{str}]
+
+  Cell identifier based on the (possibly corrected) cellular-barcode
+  sequence ({\tt CR}). The sequence may optionally be followed by a
+  dash ('-') and one or more alphanumeric characters to form an
+  identifier. The respective quality scores are (optionally) stored in
+  the {\tt CY} tag. All reads with the same {\tt CB} tag putatively
+  derive from the same cell.
+
+\item[CR:Z:\tagvalue{str}]
+  Cellular barcode.
+  Sequence bases of the cellular-barcode as reported by the sequencer (uncorrected) with the respective quality scores stored in {\tt CY}. All reads with the same {\tt CR} tag putatively derive from the same cell. 
+
+\item[CY:Z:\tagvalue{qualities+}] 
+Phred quality of the cellular-barcode sequence in the {\tt CB} (or {\tt CR}) tag. 
```

Comment:
This should only refer to the CR tag, since the CB tag contains an identifier and possibly corrected sequence for which these are not the qualities....

Reply:
this comment still stands

Reply:
CY now only refers to the CR tag and CB does not refer CY.